components/calendar/src/iso.rs

```diff
@@ -276,6 +277,52 @@ impl DateTime<Iso> {
             time: types::Time::try_new(hour, minute, second, 0)?,
         })
     }
+
+    // Minute count representation of calendars starting from 00:00:00 on Jan 1st, 1970.
+    pub fn minutes_since_local_unix_epoch(&self) -> i32 {
+        let minutes_a_hour = 60;
+        let hours_a_day = 24;
+        let minutes_a_day = minutes_a_hour * hours_a_day;
+        if let Ok(unix_epoch) = DateTime::new_iso_datetime(1970, 1, 1, 0, 0, 0) {
+            (Iso::fixed_from_iso(*self.date.inner())
+                - Iso::fixed_from_iso(*unix_epoch.date.inner()))
+                * minutes_a_day
+                + i32::from(self.time.hour.number()) * minutes_a_hour
+                + i32::from(self.time.minute.number())
+        } else {
+            0
+        }
+    }
+
+    // Convert minute count since 00:00:00 on Jan 1st, 1970 to ISO Date.
+    pub fn from_minutes_since_local_unix_epoch(
+        minute: i32,
+    ) -> Result<DateTime<Iso>, DateTimeError> {
+        let minutes_a_hour = 60;
+        let hours_a_day = 24;
+        let minutes_a_day = minutes_a_hour * hours_a_day;
+        let extra_days = minute / minutes_a_day;
+        if let Ok(unix_epoch) = DateTime::new_iso_datetime(1970, 1, 1, 0, 0, 0) {
+            let unix_epoch_days = Iso::fixed_from_iso(*unix_epoch.date.inner());
+            let date = Iso::iso_from_fixed(unix_epoch_days + extra_days);
+            match (
+                i8::try_from((minute / minutes_a_hour) % hours_a_day).ok(),
+                i8::try_from(minute % minutes_a_hour).ok(),
```

Comment:
hours_a_day/miuites_an_hour fits in an i8, you can unwrap here and leave a comment saying that the math makes it impossible

Reply:
It seems like you want me to define sth like this:
```
let minutes_a_hour: u8 = 60;
let hours_a_day: u8 = 24;
```

But since `minute` is `i32`, I need to convert the result to `i8` anyway.

Reply:
no, not at all, I was just saying that this `%` operation is guaranteed to be in range. You can simply call `... as i8` or use `i8::try_from(...).unwrap()`

Reply:
sg!